### ISSUE TITLE
1.5.3 release breaks user crontab

### ISSUE THREAD

**Issue by Polynomial-C** [cronie-crond/cronie#30]
Steps to reproduce:

```
user@linux:~> crontab -l
SHELL=/bin/bash         
                                                                               
* * * * *               echo "date1 $(date)" >> ${HOME}/temp/cronie_test
* * * * *               echo "date2 $(date)" >> ${HOME}/temp/cronie_test       
```

result with cronie-1.5.2:
```
user@linux:~> cat ~/temp/cronie_test
date1 Sun Mar 17 15:00:01 CET 2019
date2 Sun Mar 17 15:00:01 CET 2019
date2 Sun Mar 17 15:01:01 CET 2019
date1 Sun Mar 17 15:01:01 CET 2019
date2 Sun Mar 17 15:02:01 CET 2019
date1 Sun Mar 17 15:02:01 CET 2019
date2 Sun Mar 17 15:03:01 CET 2019
date1 Sun Mar 17 15:03:01 CET 2019
date2 Sun Mar 17 15:04:01 CET 2019
date1 Sun Mar 17 15:04:01 CET 2019
```

result with cronie-1.5.3:
```
user@linux:~> cat ~/temp/cronie_test
date1 Sun Mar 17 15:10:01 CET 2019
date1 Sun Mar 17 15:11:01 CET 2019
date1 Sun Mar 17 15:12:01 CET 2019
date1 Sun Mar 17 15:13:01 CET 2019
date1 Sun Mar 17 15:14:01 CET 2019
```

See also downstream bug report: https://bugs.gentoo.org/680764

**Comment by Polynomial-C** [comment]
Just to make this a bit more clean. Only the first entry of a user's crontab will be executed with cronie-1.5.3

**Comment by Polynomial-C** [comment]
Fixed in 1.5.4 release
